compiler/luci/pass/src/CircleQuantizer.cpp

```diff
@@ -69,12 +70,15 @@ class QuantizeOptionsImpl final : public luci::CircleQuantizer::Options
   const std::string param(AlgorithmParameters) const final;
   void params(AlgorithmParameters, std::vector<std::string> &) final;
   std::vector<std::string> params(AlgorithmParameters) const final;
+  void layer_params(AlgorithmParameters, std::vector<std::shared_ptr<LayerParam>> &) final;
+  std::vector<std::shared_ptr<LayerParam>> layer_params(AlgorithmParameters) const final;
   bool query(Algorithm) final;
 
 private:
   std::vector<Algorithm> _algorithms;
   std::map<AlgorithmParameters, const std::string> _algorithm_params;
   std::map<AlgorithmParameters, std::vector<std::string>> _multiple_params;
+  std::map<AlgorithmParameters, std::vector<std::shared_ptr<LayerParam>>> _multiple_layer_params;
```

Comment:
(optional) I'm not sure but just `_layer_params` without `multiple` looks okay because param"s" already means "multiple".

Reply:
_layer_params looks better. I've updated the name. Thanks.